### ISSUE TITLE
The 'Already have an account?' text on /register doesn't meet accessibility (WCAG 2.0 AA) standards for contrast.

### ISSUE THREAD

**Issue by dhicking** [tightenco/onramp#217]
![image](https://user-images.githubusercontent.com/828307/85887913-9e8a0e00-b7b6-11ea-967d-7ddde9c5165f.png)

I'd recommend making the text darker until we reach a 4.5:1 contrast ratio.

**Comment by stroudn1** [comment]
Hello! Changing the text color to #6A788A reaches 4.5:1. [Colorsafe](http://colorsafe.co/) is great to use too for creating color palettes that reach acceptable contrast. Hope this helps! 

Have a great weekend!
